src/test/suite/extension.test.ts

```diff
@@ -97,6 +97,22 @@ suite('Extension Test Suite', () => {
 		assert.strictEqual(source, undefined, 'Removing a GitSource and refreshing all views should unlist it');
 	});
 
+	test('OCI Sources are listed', async function() {
+		this.timeout(15000);
+
+		await api.shell.execWithOutput('flux create source oci podinfo --url=oci://ghcr.io/kingdonb/podinfo/deploy --tag-semver 6.1.x');
```

Comment:
We should not have this external dependency in here, (we can fix it for the next release)